x/erc20/keeper/ibc_callbacks_test.go

```diff
@@ -246,6 +247,22 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 			expErc20s:     big.NewInt(0),
 			expCoins:      coins,
 		},
+		{
+			name: "error - erc20 token not registered",
```

Comment:
in the name can you put why it won't succeed

Reply:
But it says it already - It will error because the erc20 token was not registered prior to the IBC transfer